feature-libs/cart/base/components/add-to-cart/add-to-cart.component.ts

```diff
@@ -157,20 +161,49 @@ export class AddToCartComponent implements OnInit, OnDestroy {
     }
   }
 
+  refreshLiveStockData(): void {
+    if (this.featureToggles.realTimeStockDispaly) {
+      this.currentProductService
```

Comment:
MAJOR: can we make a single call to api here instead of 2 calls?

Reply:
done